sample/src/main/kotlin/com/slack/circuit/sample/repo/PetRepository.kt

```diff
@@ -33,7 +33,7 @@ class PetRepositoryImpl @Inject constructor(private val petFinderApi: PetfinderA
   PetRepository {
   private lateinit var animals: List<Animal>
 
-  override suspend fun getAnimals(): List<Animal> {
+  override suspend fun getAnimals(forceRefresh: Boolean): List<Animal> {
     if (!this::animals.isInitialized) fetchAnimals()
```

Comment:
Do you want to implement the actual refresh in the repository? We'd need to move away from `lateinit var` in this case